README.md

```diff
@@ -42,5 +42,5 @@ Follow the steps below to save an initial state that can be loaded later for oth
         "Account": "rNGknFCRBZguXcPqC63k6xTZnonSe6ZuWt",
         "Balance": "5000000000",
    ```
-5. Copy the node's files `cp -a ~/.ziggurat/testnet/1 ~/.ziggurat/ripple_test_state`
+5. Copy the node's files `cp -a ~/.ziggurat/testnet/1 ~/.ziggurat/ripple_stateful`
```

Comment:
```suggestion
5. Copy the node's files `cp -a ~/.ziggurat/testnet/1 ~/.ziggurat/ripple/test_state`
```

Reply:
also, this should rather reference the applicable `const`

Reply:
As agreed with Niklas this change will be made in another PR.